src/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="stylesheet" href= "index.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Kalam:wght@300;400&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/1bc3a7e061.js" crossorigin="anonymous"></script>
+  <title>My to-do list</title>
+</head>
+<body>
+  <main class="container">
+    <header class="title-section">
+      <h1>Today's to-do list </h1> <i class="fa-solid fa-arrows-rotate" aria-label="Rotate icon"></i>
+    </header>
+    <section class="form-field-section">
+      <form action="submit" class="input-field">
+        <input type="text" id="input-task" class="input-task" autocomplete="off" required>
+        <label for="input-task">Add your task here...</label>
+        <input type="button" id="addBtn" value="+">
+      </form>
+    </section>
+    <section class="list-container-section">
+      <ul class="todo-list" id="todoList">
+
+      </ul>
+      <input type="button" class="clear-completed" value="Clear all completed">
+    </section>
```

Comment:
- Hey, great job with the project, everything well implemented, however, let's try to follow the laid down design as much as we can. Please, update your styling to match the given design to make your design consistent following te given design. 


*your design*
![image](https://github.com/mariegrace31/To-do-List/assets/59397159/86ee13f6-b9e8-488f-b713-6ca25e7f5fa6)


*Main design*
![image](https://github.com/mariegrace31/To-do-List/assets/59397159/40202319-c78b-432c-9a98-1d218608541d)

Reply:
Thanks @zieeco for your review. Are talking about the colors? or what?

Reply:
@mariegrace31, Yes, make your design a replica of the design template you can see.